admin/messybrainz/sql/create_functions.sql

```diff
@@ -17,7 +17,7 @@ RETURNS uuid[] AS $converted_array$
 DECLARE
     converted_array uuid[];
 BEGIN
-    SELECT array_sort(array_agg(elements)::uuid[]) || ARRAY[]::uuid[] INTO converted_array
+    SELECT public.array_sort(array_agg(elements)::uuid[]) || ARRAY[]::uuid[] INTO converted_array
```

Comment:
The array_agg here will also need to be schema qualified?

Reply:
I think so!

Reply:
array_agg is a builtin function in the pg_catalog schema, so it should be fine - I was able to get the vacuum working by only changing array_sort